spec/cypress/integration/schools/ChooseProgramme.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Induction tutors choosing programmes
+  Induction tutors should be able to choose between the Full and Core
+  induction programmes for their school
+
+  Background: 
+    Given scenario "schools/choose_programme" has been run
+
+  Scenario: Choosing Core Induction Programme
+    Given I am logged in as an "induction_coordinator"
+    Then I should be on "choose programme" page
+    
+    When I click on "accredited materials" label
+    And I click the submit button
+    Then I should be on "schools" page
+
+    When I am on "choose programme" page
+    Then I should be on "schools" page
```

Comment:
That reads a bit weird, probably because immediate redirects aren't something we wrote our cucumber steps for :/

Reply:
We could add a `I should have been redirected to` step which does the same thing (idk if there's a way to check it's actually redirected too)